common/test/acceptance/tests/discussion/test_discussion_management.py

```diff
@@ -356,11 +356,39 @@ def test_not_divided_hides_discussion_topics(self):
 
     def test_disabling_cohorts(self):
         """
-        Test that disabling cohorts hides the cohort division scheme iff it is not the selected scheme
+        Test that disabling cohorts hides the discussion management tab if 1 or less enrollment track
         (even without reloading the page).
         """
-        # TODO: will be added as part of AJ's work.
-        pass
+        self.disable_cohorting(self.course_fixture)
+        self.instructor_dashboard_page.visit()
+        self.assertFalse(self.instructor_dashboard_page.is_discussion_management_visible())
+
+    def test_disabling_cohorts_with_enrollment_tracks(self):
+        """
+        Test that disabling cohorts does not hide the discussion tab when there are more than one enrollment track, but
```

Comment:
And I don't see a test for the cohort radio button hiding after cohorts are disabled.